fix(curriculum): add word nesting & comments to nutrition-label-step-43

marktlinn opened the pull request:
Add comments to code to clear up confusion about where user's should actually be writing their code, as this has been the cause of problems and frustration for some. Also improve the description to incorporate the use of the phrasing 'nesting' in a clear and understandable manner as per suggestions in issue #49017

Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [ X ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ X ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ X ] My pull request targets the `main` branch of freeCodeCamp.
- [  X ] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

fix #49017

<!-- Feel free to add any additional description of changes below this line -->

Improve the description to incorporate the use of the phrasing 'nesting' in a clear and understandable manner as per suggestions in issue #49017. Also, added comments to code to clear up confusion about where user's should actually be writing their code, as this has been the cause of problems and frustration for some.

marktlinn committed:
fix: clarify  description wording & add comments

Add comments to code to clear up confusion about where user's should actually be writing their code, as this has been the cause of problems and frustration for some. Also improve the description to incorporate the use of the phrasing 'nesting' in a clear and understandable manner as per suggestions in issue #49017

marktlinn committed:
fix (comment): rewrite comment by repo rules

Re-wrote the comment in order to comply with the repo dictionary for transalation puposes.

Sboonny commented on curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f6b2d164f81809efd9bdc.md:
```diff
@@ -95,6 +95,9 @@ assert(document.querySelector('.daily-value.small-text')?.lastElementChild?.inne
     <div class="daily-value small-text">
       <p class="bold right">% Daily Value *</p>
       <div class="divider"></div>
+      <!--Add your code below this line-->
+      
+      <!--Add your code above this line-->
```
```suggestion
```

Sboonny commented on curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f6b2d164f81809efd9bdc.md:
```diff
@@ -7,7 +7,7 @@ dashedName: step-43
 # --description--
 
-After your last `.divider` element, create a `p` element and give it the text `Total Fat 8g 10%`. Wrap `Total Fat` in a `span` element with the `class` set to `bold`. Wrap `10%` in another `span` element with the `class` set to `bold`. Also wrap the words `Total Fat 8g` in an additional `span` element for alignment.
+After your last `.divider` element, create a `p` element and give it the text `Total Fat 8g 10%`. Wrap `Total Fat` in a `span` element with the `class` set to `bold`. Wrap `10%` in another `span` element with the `class` set to `bold`. Finally wrap both the `span` element `Total Fat` and the text `8g` with a `span`, so that they are nested together inside a `span` element.
```
```suggestion
After your last `.divider` element, create a `p` element and give it the text `Total Fat 8g 10%`. Wrap `Total Fat` in a `span` element with the `class` set to `bold`. Wrap `10%` in another `span` element with the `class` set to `bold`. Finally, nest these two `span` elements in an additional `span` element for alignment.
```

Sboonny reviewed (changes requested):
To solve this issue:
- change the wording to what agreed upon in the issue.
- change step 53 as well.

---

Can you edit the PR title to reflect on our conviction, this isn't how you add scope

marktlinn committed:
Update curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f6b2d164f81809efd9bdc.md

wording altered to exactly match issue's suggestion

Co-authored-by: Muhammed Mustafa <muhammed@freecodecamp.org>

marktlinn committed:
Update curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f6b2d164f81809efd9bdc.md

Co-authored-by: Muhammed Mustafa <muhammed@freecodecamp.org>

marksist300 commented:
> To solve this issue:
> 
>     * change the wording to what agreed upon in the issue.
> 
>     * change step 53 as well.
> 
> 
> Can you edit the PR title to reflect on our conviction, this isn't how you add scope

Yes, of course and thanks for the comment. I've made the changes, I hope all is ok now. Let me know if there's anything else I can change or improve 🙂. I'll take a look at step now 53 too.

marktlinn committed:
fix curriculum: clarify desc for nutrional-label, step-53

Update description wording to use the phrase nest to make it simpler and easier to follower for users

marksist300 commented:
> To solve this issue:
> 
>     * change the wording to what agreed upon in the issue.
> 
>     * change step 53 as well.
> 
> 
> Can you edit the PR title to reflect on our conviction, this isn't how you add scope

@Sboonny Done, added a similar description to step 53 👍🏻

Sboonny reviewed (approved):
LGTM 🎉, thank you for your contribution to the page! 👍.

ShaunSHamilton commented on curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f6b2d164f81809efd9bdc.md:
```diff
@@ -7,7 +7,7 @@ dashedName: step-43
 # --description--
 
-After your last `.divider` element, create a `p` element and give it the text `Total Fat 8g 10%`. Wrap `Total Fat` in a `span` element with the `class` set to `bold`. Wrap `10%` in another `span` element with the `class` set to `bold`. Also wrap the words `Total Fat 8g` in an additional `span` element for alignment.
+After your last `.divider` element, create a `p` element and give it the text `Total Fat 8g 10%`. Wrap `Total Fat` in a `span` element with the `class` set to `bold`. Wrap `10%` in another `span` element with the `class` set to `bold`. Finally, nest these two `span` elements in an additional `span` element for alignment.
```
The desired outcome is:

```html
<p>
  <span>
    <span class="bold">Total Fat</span> 8g
  </span> 
  <span class="bold">10%</span>
</p>
```

These instructions suggest:

```html
<p>
  <span>
    <span class="bold">Total Fat</span> 8g <span class="bold">10%</span>
  </span>
</p>
```

marksist300 replied:
@ShaunSHamilton Yes, you're right! I have tried to rectify it. Hopefully the wording is more accurate and helpful now. Let me know if there's anything I can do to improve it. Thanks for the feedback! 🙂

ShaunSHamilton reviewed (changes requested):
Step 43 needs more clarification.

marktlinn committed:
refactor: update wording to clarify instruction

ShaunSHamilton reviewed (approved):
This LGTM 👍 

Thank you for making that change, and for this contribution 🙂
